test/image/mocks/limit_select_any.json

```diff
@@ -0,0 +1,15 @@
+{
+  "data": [
+    { "x": [1,1,1,1,1,2,2,2,3,3], "type": "histogram" },
+    { "x": [1,2,3], "y":[-1,-2,-3], "type": "bar" },
+    { "x": [1,2,3], "y":[-1,-2,-3], "type": "bar", "yaxis":"y2" },
+    { "x": [1,1,1,1,1,2,2,2,3,3], "type": "histogram", "yaxis":"y2" }
+  ],
+  "layout": {
+    "yaxis": { "domain": [0,0.49] },
+    "yaxis2": { "domain": [0.51, 1] },
+    "showlegend": false,
+    "dragmode": "select",
+    "selectdirection": "any"
```

Comment:
I don't think we need to make image test mocks for `selectdirection`, since it has no effect on the static images. For the jasmine tests, if you don't like the mock ([14](https://github.com/plotly/plotly.js/blob/master/test/image/mocks/14.json)) that we use in the neighboring tests, you can use any other predefined mock, or just make a new plot explicitly in the test, something like:
```js
Plotly.newPlot(gd, [
    {x: [1,1,1,1,1,2,2,2,3,3], type: 'histogram'},
    {x: [1,2,3], y:[-1,-2,-3], type: 'bar'}
], {
    showlegend: false,
    dragmode: 'select'
})
.then(function() {
    expect(gd.fullLayout.selectdirection).toBe('any');

    // do a drag and check that the selection is correct for the default `selectdirection`

    return Plotly.relayout(gd, {selectdirection: 'h'});
})
.then(function() {
    // do probably the SAME drag and check that the selection is correct for the `selectdirection='h'`

    return Plotly.relayout(gd, {selectdirection: 'v'});
})
.then(function() {
    // ...
})
.catch(fail) // hmm, we should change the name of this import to failTest, since fail is already a jasmine global, for sync failure, we're overriding it here with our async version (we've already done this in other places)
.then(done);
```